src/pages/login.js

```diff
@@ -1,5 +1,88 @@
+import Background from "../assets/universe_background.jpeg";
+import Logo from "../assets/spacecenter1.svg";
+import { useNavigate } from "react-router-dom";
+import { post } from "../Utils";
+
 function Login() {
-  return <h1>Login</h1>;
+  const navigate = useNavigate();
+
+  const current_session = {
```

Comment:
Just declaring an object like this inside a React component is not very useful, as it basically will be recreated on every render (which happens a lot). If you want the value to persist across renders, you need to store it with `useState` or `useMemo`, though in this case, you probably don't need it at all.